CHANGELOG.md

```diff
@@ -4,10 +4,60 @@ All notable changes to this project will be documented in this file.
+- Added numerous additional functions to sniff for to the `WordPress.VIP.RestrictedFunctions` sniff as per the VIP guidelines.
+- Upped the `posts_per_page` limit from 50 to 100 `WordPress.VIP.PostsPerPage` sniff as per the VIP guidelines.
+- Added `__debugInfo` to the magic method whitelist for functions starting with double underscore in the `WordPress.NamingConventions.ValidFunctionName` sniff.
+- Added `cat_ID` to the whitelisted exceptions for the `WordPress.NamingConventions.ValidVariableName` sniff.
+- An error will now be thrown for non-magic functions using a double underscore prefix - `WordPress.NamingConventions.ValidFunctionName` sniff.
+- The `WordPress.Arrays.ArrayAssignmentRestrictions`, `WordPress.Functions.FunctionRestrictions`, `WordPress.Variables.VariableRestrictions` sniffs weren't in actual fact sniffs, but parent classes for child sniffs. These have now all been turned into proper abstract parent classes and moved to the main `WordPress` directory.
+- The array provided to `AbstractFunctionRestrictions` can now take a `whitelist` key to whitelist select functions when blocking a group of functions by function prefix.
+- Updated installation instructions in the readme.
+
+### Removed
+- `Squiz.Functions.FunctionDeclarationArgumentSpacing.SpacingBeforeClose` from the `WordPress-Core` standard (was causing duplicate messages for the same issue).
+- `Squiz.Commenting.FunctionComment.ScalarTypeHintMissing`, `Squiz.Commenting.InlineComment.NotCapital` from the `WordPress-Docs` standard.
+- Removed the sniffing for `get_pages()` from the `WordPress.VIP.RestrictedFunctions` sniff as per the VIP guidelines.
+- Removed the sniffing for the `extract()` and POSIX functions from the `WordPress.VIP.RestrictedFunctions` sniff as these now have their own sniffs.
+
+### Fixed
+- Error message precision for the `WordPress.NamingConventions.ValidVariableName` sniff.
+- Bug in the `WordPress.WhiteSpace.ControlStructureSpacing.BlankLineAfterEnd` sniff which was incorrectly being triggered on last method of class.
+- Function name sniffs based on the `AbstractFunctionRestrictions` parent class will now do a case-insensitive function name comparison.
+- Function name sniffs in the `WordPress.PHP.DiscouragedFunctions` sniff will now do a case-insensitive function name comparison.
+- Whitelist comments directly followed by a PHP closing tag were not being recognized.
+- Some PHP Magic constants were not recognized by the `WordPress.XSS.EscapeOutput` sniff.
+- An error message suggesting Camel caps rather than the intended snake case format in the `WordPress.NamingConventions.ValidFunctionName` sniff.
+- `WordPress.WhiteSpace.ControlStructureSpacing` should no longer throw error notices during live code review.
+- Errors will be no longer be thrown for methods not complying with the naming conventions when the class extends a parent class or implements an interface - `WordPress.NamingConventions.ValidFunctionName` sniff.
+
+### Housekeeping
```

Comment:
I think these three could all go under the Changed section, instead of introducing a new section.